test/val/val_type_unique_test.cpp

```diff
@@ -238,4 +238,57 @@ OpMemoryModel Physical32 OpenCL
               Not(HasSubstr(GetErrorString(SpvOpTypeVoid))));
 }
 
+TEST_F(ValidateTypeUnique, PointerTypesSameArrayStrideNoExtension) {
+  string str = R"(
+OpCapability Shader
+OpCapability Linkage
+OpMemoryModel Logical GLSL450
+OpDecorate %ptr1 ArrayStride 4
+OpDecorate %ptr2 ArrayStride 8
+%u32 = OpTypeInt 32 0
+%ptr1 = OpTypePointer Input %u32
+%ptr2 = OpTypePointer Input %u32
+)";
+  CompileSuccessfully(str.c_str());
+  ASSERT_EQ(SPV_SUCCESS, ValidateInstructions());
+  EXPECT_THAT(getDiagnosticString(),
+              HasSubstr(GetErrorString(SpvOpTypePointer)));
```

Comment:
I'm confused.  The validation succeeds, but generates an error string?

Does this test even pass?

Reply:
It logs a warning but returns SPV_SUCCESS.